specs/BrowserProcessExited.md

```diff
@@ -53,6 +53,18 @@ class AppWindow {
 
   wil::com_ptr<ICoreWebView2Controller> m_controller;
   EventRegistrationToken m_browserExitedEventToken = {};
+  UINT32 m_newestBrowserPid = 0;
+}
+
+HRESULT AppWindow::OnCreateCoreWebView2ControllerCompleted(HRESULT result, ICoreWebView2Controller* controller)
+{
+  // ...
+
+  // Save PID of the browser process serving last WebView created from our
+  // CoreWebView2Environment.
+  CHECK_FAILURE(m_webView->get_BrowserProcessId(&m_newestBrowserPid));
```

Comment:
What happens if you call `get_BrowserProcessId` too soon (before the WebView browser process has been created) or too late (after it crashed)?

Reply:
If the creation completed handler ([ICoreWebView2CreateCoreWebView2ControllerCompletedHandler](https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2createcorewebview2controllercompletedhandler?view=webview2-1.0.818.41)) was called with an `S_OK` HRESULT, the connection has already been established to a running browser process. The full sample app has an `S_OK` check before this call, I'll update to reflect that here.

On browser process crash, the webview moves to its closed state and the call will fail. In this case, there will be a [ProcessFailed](https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2?view=webview2-1.0.818.41#add_processfailed) event raised for the crash.

Reply:
Is the transition to the Closed state synchronous or asynchronous? In other words, is it possible for the app to observe the Closed state before it receives the ProcessFailed event? If synchronous, then the "too late" failure is avoidable. if asynchronous, then apps cannot protect against the "too-late" case.

Reply:
Synchronous.

Reply:
Sounds good. Please make this clear in the documentation as appropriate.

Reply:
Since this is about `get_BrowserProcessId` which is from a different interface, I don't think this API is the right place to document that. I added a comment to the sample code, though.